website/package.json

```diff
@@ -16,7 +16,8 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids"
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "test": "NODE_ENV=test node --trace-warnings ../node_modules/.bin/jest remote-content"
```

Comment:
Ok, this is very cheap of me, I could have added jest as a dependency inside website, but this works well for local testing. Not intended to be used in CI, it just gives me a faster feedback loop than building the contents. I can remove it but I wanted to keep the tests for the review process

Reply:
yes, I'm not using it during build time, so right now it's a handy method to test locally, but the tests are not executed on CI, maybe we should? For now, from no test to manual tests I'm ok.

Reply:
Right. Yeah, I think we could do it in CI, but we'd probably need as local Jest for that , then, yeah? We haven't really had much in the way of tests in the docs before, so we'd have to set it up, but it sounds smart. But then again, how often do we change this? About once every two years? Still, be nice to have it checked when it does change